sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -149,4 +157,79 @@ export class InstitutionController extends BaseController {
   async getUserTypesAndRoles(): Promise<InstitutionUserTypeAndRoleResponseDto> {
     return this.institutionService.getUserTypesAndRoles();
   }
+
+  @Get("/user/:userName")
+  async getInstitutionLocationByUserName(
+    @Param("userName") userName: string,
+  ): Promise<InstitutionUser> {
+    // Get institutionUser
+    const institutionUser = await this.institutionService.getInstitutionUserByUserName(
+      userName,
+    );
+    if (!institutionUser) {
+      throw new UnprocessableEntityException("Not able to retrieve the institution location user.");
+    } else if (!institutionUser.user.isActive){
```

Comment:
We can leave this check for now, but this should be handled globally by the API.
I am planning to introduce this with the PR that I am working on.